src/OrchardCore.Modules/OrchardCore.Media/GraphQL/MediaAssetQuery.cs

```diff
@@ -10,11 +10,12 @@
 
 namespace OrchardCore.Media.GraphQL;
 
-public class MediaAssetQuery : ISchemaBuilder
+public sealed class MediaAssetQuery : ISchemaBuilder
 {
-    protected readonly IStringLocalizer S;
     private readonly GraphQLContentOptions _graphQLContentOptions;
 
+    internal IStringLocalizer S;
```

Comment:
Can't these be private, or does the PoExtractor depend on those fields being internal?